gensim/models/ldamodel.py

```diff
@@ -354,25 +373,25 @@ def init_dir_prior(self, prior, name):
 
         if isinstance(prior, six.string_types):
             if prior == 'symmetric':
-                logger.info("using symmetric %s at %s", name, 1.0 / prior_shape)
-                init_prior = np.asarray([1.0 / self.num_topics for i in xrange(prior_shape)])
+                logger.info("using symmetric %s at %s", name, 1.0 / prior_shape) #TODO: prior_shape?
```

Comment:
I have feeling that it should be

```
"using symmetric %s at %s", name, 1.0 / self.num_topics
```

am I right? @menshikh-iv

Reply:
Yes @xelez, you are correct!